src/tie_breaks/individual.py

```diff
@@ -279,6 +283,8 @@ def buchholz(
             else:
                 scores.append(dummy_points)
             continue
+        if pairing.opponent_id is None:
+            continue
```

Comment:
If we get there, we should have an `opponent_id`, that's why I didn't check.
When does this break?